internal/cri/opts/spec_opts.go

```diff
@@ -353,6 +353,7 @@ func WithPodNamespaces(config *runtime.LinuxContainerSecurityContext, sandboxPid
 		oci.WithLinuxNamespace(runtimespec.LinuxNamespace{Type: runtimespec.NetworkNamespace, Path: GetNetworkNamespace(sandboxPid)}),
 		oci.WithLinuxNamespace(runtimespec.LinuxNamespace{Type: runtimespec.IPCNamespace, Path: GetIPCNamespace(sandboxPid)}),
 		oci.WithLinuxNamespace(runtimespec.LinuxNamespace{Type: runtimespec.UTSNamespace, Path: GetUTSNamespace(sandboxPid)}),
+		oci.WithLinuxNamespace(runtimespec.LinuxNamespace{Type: runtimespec.TimeNamespace, Path: GetTimeNamespace(sandboxPid)}),
```

Comment:
Needs a KEP

Reply:
Agree with @AkihiroSuda. This should not be enabled by default without a KEP.